ui/src/components/form-view/QasFormView.vue

```diff
@@ -213,6 +213,10 @@ export default {
   watch: {
     isSubmitting (value) {
       this.$emit('update:submitting', value)
+    },
+
+    '$route.path' () {
+      this.ignoreRouterGuard = false
```

Comment:
e se `use-dialog-on-unsaved-changes` for false?

Reply:
Respondido pelo discord